File: app/src/main/java/com/utynote/components/search/model/geojson/entities/Query.java

package com.utynote.components.search.model.geojson.entities;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Query {

    @SerializedName("text")
    @Expose
    public String text;
    @SerializedName("size")
    @Expose
    public Integer size;
    @SerializedName("private")
    @Expose
    public Boolean _private;
    @SerializedName("lang")
    @Expose
    public Lang lang;
    @SerializedName("querySize")
    @Expose
    public Integer querySize;

}

File: app/src/main/java/com/utynote/components/search/data/geojson/entities/Query.kt
package com.utynote.components.search.data.geojson.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Query {

    @SerializedName("text")
    @Expose
    var text: String? = null
    @SerializedName("size")
    @Expose
    var size: Int? = null
    @SerializedName("private")
    @Expose
    var _private: Boolean? = null
    @SerializedName("lang")
    @Expose
    var lang: Lang? = null
    @SerializedName("querySize")
    @Expose
    var querySize: Int? = null

}
